plugins/GSdx/GSRendererHW.cpp

```diff
@@ -55,7 +55,8 @@ void GSRendererHW::SetScaling()
 	// Framebuffer width is always a multiple of 64 so at certain cases it can't cover some weird width values.
 	// 480P , 576P use width as 720 which is not referencable by FBW * 64. so it produces 704 ( the closest value multiple by 64).
 	// In such cases, let's just use the CRTC width.
-	int fb_width = max((int)m_context->FRAME.FBW * 64, crtc_width);
+	auto search_fb_width = minmax({(int)m_context->FRAME.FBW * 64, crtc_width , 512});
+	int fb_width = search_fb_width.second;
```

Comment:
done, though I don't see the advantage on having it as a member variable. I liked the old code better :\